.github/workflows/main.yml

```diff
@@ -350,7 +367,7 @@ jobs:
               ninja_arg: "",
               openssl: "yes",
               openssl_arg: "linux-x86",
-              C: "yes",
+              C: "no",
```

Comment:
You can enable this C test, here it is for the x86 architecture

Reply:
My bad, I wanted to turn it off for for `linux-armv5-uclibc` not sure how I managed to turn it off for `linux-x86`